docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -0,0 +1,174 @@
+
+|==============================================
+// end::update-exception-item-request-body[]
+
+
+
+// tag::api-id-or-itemid-query-params[]
+The URL query must include one of the following:
+
+* `id` - `/api/exception_lists/items?id=<id>`
+* `item_id` - `/api/exception_lists/items?item_id=<item_id>`
+// end::api-id-or-itemid-query-params[]
+
+
+// tag::find-exception-items-query-params[]
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`list_id` |String a|Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
+|`namespace_type` |String a|Must be set to `agnostic`. |Yes
+|`page` |Integer |The page number to return. |No
+|`per_page` |Integer |The number of items to return per page. |No
+|`sort_field` |String |Determines which field is used to sort the results. |No
+|`sort_order` |String a|Determines the sort order, which can be `desc` or `asc`. |No
+|`filter` |String |A {kibana-ref}/kuery-query.html[{kib} Query Language (KQL)] string to filter the results down. |No
+
+|==============================================
+// end::find-exception-items-query-params[]
+
+
+
+// tag::endpoint-exceptions-value-types-info-section[]
+==== Value types
+
+The following describes the types that can be defined when using the xref:create-item-api[create] or xref:update-item-api[update] {endpoint-artifact-name} APIs:
+
+[#exception-item-comment-schema]
+===== `comment` object schema
+
+Comments are JSON objects containing the following structure:
+
+[source,json]
+---------
+{
+    "comment": "some comment here"
+}
+---------
+
+When used with the update API, existing comments can be updated by using their associated `id`, while any comment without the `id` attribute will be added as a new comment:
+
+[source,json]
+---------
+{
+    "comment": "some comment here - updated",
+    "id": "1078cf59-5893-4143-acf7-40a40af16bee"
+}
+---------
+
+[#exception-item-os-types]
+===== `os_types` values
+
+The following are the valid OS types that can be used when creating {endpoint-artifact-name}:
+
+* `windows`
+* `linux`
+* `macos`
+
+
+[#exception-item-scope-assignment-tags]
+===== Scope assignment
+
+The {endpoint-artifact-name} can be assigned globally across all endpoint policies, or assigned to specific policies. The assignment of a {endpoint-artifact-name-single} is done by defining one or more tags with a prefix of `policy:`. Note that the {endpoint-artifact-name-single} can be either "global" or "per policy", but not *both*. The following tags are available for use in order to control the assignment scope:
+
+* `policy:all` : {endpoint-artifact-name-single} is global to all policies. If used, no other `policy:` tag is allowed.
+* `policy:<endpoint-policy-id>` : {endpoint-artifact-name-single} is assigned to a policy. Multiple "per policy" tags can be used to associate the {endpoint-artifact-name-single} to multiple policies.
```

Comment:
A couple more capitalization issues, as these render lowercase:

<img width="500" alt="image" src="https://user-images.githubusercontent.com/87339667/156659301-ad4097a1-c246-40f8-996c-3090341df9db.png">

Maybe adding "The" will fix?
```suggestion
* `policy:all` : The {endpoint-artifact-name-single} is global to all policies. If used, no other `policy:` tag is allowed.
* `policy:<endpoint-policy-id>` : The {endpoint-artifact-name-single} is assigned to a policy. Multiple "per policy" tags can be used to associate the {endpoint-artifact-name-single} to multiple policies.
```

Reply:
@joepeeples  -= I ended up creating two more variables (see below) and will use these in the docs

```
:endpoint-artifact-name-sentence-start:         Trusted applications
:endpoint-artifact-name-single-sentence-start:  Trusted application
```